### PR TITLE
added new script as Secured Chat machine

### PR DESCRIPTION
**Pull request by Dr-blue-cyber**
# added new script in script folder with filename 
main.py and README.md 

Fixes #(issue number)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ X] New Script (non-breaking change which adds a new script to the repository)
- [ ] This change requires a documentation update

## Has This Been Tested?

- [ X] I confirm that I have tested my script locally and verified that it works as intended.

## Screenshot (if applicable)

Screenshot 2021-10-04 024756
## Checklist:

- [ X ] I have gone through the contributing guidelines before making this Pull Request.
- [ x ] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by Dr-blue-cyber** [comment]
can explain why this is happend ???

**Comment by belelaritra** [comment]
> @AcousticCh @OBITORASU can you guys check this PR ???
> 
> and anybody can explain why this ALL Checks have Failed so I can improve it

1. Install `black` in your local from [here](https://pypi.org/project/black/)
>or, use this command 
```
 pip install black 
```
2. Run `black .` command inside your folder Named : <b> Secure chat machine</b>.
>or, in your case 
```
black main.py
```

**Comment by Dr-blue-cyber** [comment]
> > @AcousticCh @OBITORASU can you guys check this PR ???
> > and anybody can explain why this ALL Checks have Failed so I can improve it
> 
> 1. Install `black` in your local from [here](https://pypi.org/project/black/)
> 
> > or, use this command
> 
> ```
>  pip install black 
> ```
> 
> 1. Run `black .` command inside your folder Named :  Secure chat machine.
> 
> > or, in your case
> 
> ```
> black main.py
> ```

thanks you so much @belelaritra  I did it  as you mentioned 




here you can see


but then also it showing "ALL CHECKS ARE FAILED" 
so i have commit again??

 
Screenshot 2021-10-05 073134

**Comment by AcousticCh** [comment]
When I had an issue of a PR refusing to pass checks it was because I removed and reinstalled my kernel and os and forgot to reset my global config user and email options therefore GitHub never received a git Id of the repo that pushed the PR @Dr-blue-cyber

**Comment by AcousticCh** [comment]
@Dr-blue-cyber but your check is a build check, maybe some dependency is missing from the directory if your file is in its own directory or if your dir had more than one level a dependency could be in the wrong dir

**Comment by belelaritra** [comment]
> 
> thanks you so much @belelaritra I did it as you mentioned
> 
> here you can see
> 
> but then also it showing "ALL CHECKS ARE FAILED" so i have commit again??
> 

@Dr-blue-cyber , No need to create another [PR](#182). 
Just commit that reformatted file inside your [`main`](https://github.com/Dr-blue-cyber/General-Purpose-Scripts/tree/main) branch.

**Comment by OBITORASU** [comment]
> 
> 
> @Dr-blue-cyber but your check is a build check, maybe some dependency is missing from the directory if your file is in its own directory or if your dir had more than one level a dependency could be in the wrong dir

The name of the workflow is a bit misleading but his issue is linting nothing else.

File: scripts/Secured chat machine/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<h1 align="center"> Chat Machine  </h1>
+
+<h2 align="right">
+
+![python](https://forthebadge.com/images/badges/made-with-python.svg)
+
+</h2>
+
+This is WebAPP which provides Secured Chat messaging feaure with Encryption ans decyption security tool.
```
Comment (OBITORASU) [inline scripts/Secured chat machine/README.md:9]: This is not a WebAPP this is a GUI application run locally fix this in readme.

File: scripts/Secured chat machine/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+<h2 align="right">
+
+![python](https://forthebadge.com/images/badges/made-with-python.svg)
+
+</h2>
+
+This is WebAPP which provides Secured Chat messaging feaure with Encryption ans decyption security tool. 
+
+Base64 encryptions are scheme used in encryptions and decryption
```
Comment (OBITORASU) [inline scripts/Secured chat machine/README.md:11]: Base64 is not an encryption schema it is an encoding fix this as well.

File: scripts/Secured chat machine/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## __ABOUT__
+**ChatMachine** is a only suggest how can we achieve our End-to-End encyotion schems thats leads to secures messaging in python application that helps user to maintain our privacy from hackers and thirs party applications
```
Comment (OBITORASU) [inline scripts/Secured chat machine/README.md:15]: Fix the typos on this line. Also this is not end-to-end encryption fix that as well.

**Pull request closed without merging**